src/apps/annotation-image/LeftDrawer/LeftDrawer.css

```diff
@@ -14,6 +14,10 @@
   height: 100%;
 }
 
+.ta-left-drawer aside .tablist {
```

Comment:
Annotation view: I noticed that when the left sidebar was scrollable, the sidebar contents would show underneath the tablist. Made that fix while I was at it.